docs/USAGE.md

```diff
@@ -2,37 +2,95 @@
 
-You will also need to modify the [``ove/packages/ove-core/src/client/Spaces.json``](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file to define a new ``space`` in OVE core.
+## Setting up OVE
 
-Note that all services need to hosted somewhere that is accessible from the computers connected to the monitors that will be used in the display.
+By default, OVE provides two `spaces`, that are useful for trying out some of the [**OVE Apps**](https://github.com/ove/ove-apps) with sample content. These are:
 
-## Creating content
+* `LocalNine` - This `space` contains nine `clients` arranged in `3 x 3` configuration. Each `client` has a dimension of `1440 x 808` pixels and the total `space` has a dimension of `4320 x 2424` pixels.
+* `LocalFour` - This `space` contains four `clients` arranged in `2 x 2` configuration. Each `client` has a dimension of `1440 x 808` pixels and the total `space` has a dimension of `2880 x 1616` pixels.
 
-When designing content to be displayed in an OVE environment, be aware of the [potential pitfalls](./PITFALLS.md).
+You will need to modify the [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file and introduce a new `space` in OVE, before it can be used in a production setting.
 
 ## Launching browsers
 
-On each monitor that makes up the display you will need to launch a browser window, and open the correct URL in each. This has the form: ``http://<ove-hostname>:<ove-port>/view.html?oveViewId=<space>-{id}``
-where ``{id}`` is the index of the ``client`` associated with the browser window in the space's definition in ``Spaces.json``.
+In order to use OVE, you will need to open the URLs of the `clients` in a web browser. Each URL has the form: `http://OVE_CORE_HOST:PORT/view.html?oveViewId={SPACE}-{ID}`. The values for `OVE_CORE_HOST` and `PORT` must be set as explained in the [OVE Installation Guide](./INSTALLATION.md#running-ove). The value for `SPACE` would be the name of the `space` used, such as `LocalNine`, `LocalFour`, or a name of a new `space` that has been defined in the [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file. The value for `ID` is the index of the `client` associated with the browser window, in the definition of the `space` in [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
+
+It is recommended to use OVE with Google Chrome, as this is the web browser used for development and in production at the [Data Science Institute](http://www.imperial.ac.uk/data-science/). However, it should also be compatible with other modern web browsers: if you encounter any browser-specific bugs please [report them as an Issue](https://github.com/ove/ove-apps/issues).
+
+To test OVE functionality, you can simply open four web browser windows (or tabs) with the following URLs to create the `LocalFour` space:
```

Comment:
Is this *creating* the space?

Reply:
Good question: we need a better word here. How about `load`?

Reply:
I have rewritten as `corresponding to`. It means neither create or load since `space` is a hardware concept.